packages/extension-vscode/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "activationEvents": [
+        "onLanguage:css",
+        "onLanguage:html",
+        "onLanguage:javascript",
+        "onLanguage:json",
+        "onLanguage:jsonc"
+    ],
+    "ava": {
+        "failFast": false,
+        "files": [
+            "dist/tests/**/*.js"
+        ],
+        "timeout": "1m"
+    },
+    "dependencies": {
+        "vscode": "^1.1.21",
+        "vscode-languageclient": "^5.0.1",
+        "vscode-languageserver": "^5.0.3"
+    },
+    "description": "Run webhint in Visual Studio Code.",
+    "devDependencies": {
+        "ava": "^0.25.0",
+        "cpx": "^1.5.0",
+        "eslint": "^5.6.0",
+        "eslint-plugin-import": "^2.14.0",
+        "eslint-plugin-markdown": "^1.0.0-beta.7",
+        "eslint-plugin-typescript": "^0.12.0",
+        "hint": "^3.4.0",
+        "markdownlint-cli": "^0.13.0",
+        "npm-link-check": "^2.0.0",
+        "npm-run-all": "^4.1.2",
+        "nyc": "^13.0.1",
+        "rimraf": "^2.6.2",
+        "typescript": "^3.0.3",
+        "typescript-eslint-parser": "^18.0.0"
+    },
+    "engines": {
+        "node": ">=8.0.0",
+        "vscode": "^1.26.0"
+    },
+    "homepage": "https://webhint.io/",
+    "icon": "icon.png",
```

Comment:
Optimized a bit more:

![icon](https://user-images.githubusercontent.com/1223565/45704267-4e830f80-bb2b-11e8-890c-638ddcd4baa6.png)

Reply:
Cool, can I just download and update the PR with this one then?

Reply:
@antross Yes.

Reply:
Hmm... actually this has exactly the same file size as the current `icon.png`. What's more optimized?

Reply:
Ignore that, operator error... I see the size difference now.

Reply:
👍